backend/src/main/kotlin/be/osoc/team1/backend/entities/Communication.kt

```diff
@@ -20,8 +21,13 @@ class Communication(
     val type: CommunicationTypeEnum,
     @JsonIgnore
     @NotBlank
-    val edition: String = ""
+    val edition: String = "",
+    @JsonIgnore
+    @OneToOne
+    val student: Student

```

Comment:
A student can have more than one communication. Also, this field won't be send when requesting communications right (just making sure)?